### PR TITLE
Add a link about parrot suspension

### PR DESCRIPTION
**Pull request by perlpilot**

### CODE REVIEW

File: perl6.html.markdown [hunk 1]
```diff
@@ -7,11 +7,13 @@ contributors:
     - ["Nami-Doc", "http://github.com/Nami-Doc"]
 ---
 
-Perl 6 is a highly capable, feature-rich programming language made for the
-upcoming hundred years.
+Perl 6 is a highly capable, feature-rich programming language made for at 
+least the next hundred years.
```
Comment (vendethiel) [inline perl6.html.markdown]: That's taken from perl6.org iirc :)

**Comment by geoffliu** [comment]
@perlpilot Looks like mentions of Parrot has already been removed, but I think it's worth mentioning Raduko. Anyway, let's get this PR merged, if you'd like. It's behind master right now, could you update?

**Comment by perlpilot** [comment]
@geoffliu Updated if you still want to merge.

**Comment by vendethiel** [comment]
Thanks!

**Pull request merged**